WordPress/gutenberg
# Save panel - decouple Site Entity items for individual saving.

## Description

Aims to solve #29462 (alternative to #30779) This PR explores decoupling the site entity's edits and modularizing them in the save panel, allowing for individual checkbox selection of each edited item:

![Screen Shot 2021-04-13 at 3 22 19 PM](https://user-images.githubusercontent.com/28742426/114609818-320ccc80-9c6d-11eb-9c2f-2dd01929e252.png)

Previously, any combination of these items would fall under the same checkbox with an obscure (actual site title) label and they all would have to be saved (or not saved) together:
![image](https://user-images.githubusercontent.com/28742426/114610271-b4958c00-9c6d-11eb-89e2-ee0f035e9e4c.png)


## How has this been tested?



* Edit the Site Title, Site Tagline, and Site Logo blocks.
* Open the save panel.
* Verify items are separated under their own checkbox.
* Verify items save individually as expected.

## Screenshots 

## Types of changes





## Checklist:
